app/views/home/index.html.erb

```diff
@@ -1,10 +1,296 @@
+        <h3 class="font-weight-light">Deploy</h3>
+        <!-- Text -->
+        <p class="text-muted mb-0">Deploy to multiple locations<br/>with a single command.</p>
+      </div>
+    </div> <!-- / .row -->
+      <!-- / .row -->
+  </div> <!-- / .container -->
+</section>
+<section class="py-8 pt-md-11 pb-md-12 bg-light p-3 p-md-5">
+  <div class="container">
+    <div class="row justify-content-center">
+      <div class="col-12 col-md-10 col-lg-8 text-center">      
+        <!-- Preheading -->
+        <h6 class="text-uppercase text-success font-weight-bold">
+          All in one solution
+        </h6>
+        <!-- Heading -->
+        <h1 class="font-weight-bold">For Side/Fun Projects<br/>or <span class="text-secondary">Growing Companies</span></h1>
+        <!-- Text -->
+        <div class="row text-gray-500 mt-5 mb-7 mb-md-9">
+          <div class="col-lg-4">
+            <img src="/images/web-hosting/svg/008-setting.svg" width="140"/>
+            <h2>Hands Free.</h2>
+            <p>No need to manage servers manually. Just keep on building your projects.</p>
+          </div><!-- /.col-lg-4 -->
+          <div class="col-lg-4">
+            <img src="/images/web-hosting/svg/032-diagram.svg" width="140"/>
+            <h2>Fair Pricing.</h2>
+            <p>No subscription billing. You are billed only when your instances are up.</p>
+          </div><!-- /.col-lg-4 -->
+          <div class="col-lg-4">
+            <img src="/images/web-hosting/svg/037-website.svg" width="140"/>
+            <h2>Low Cost.</h2>
+            <p>Tiny instances (as low as as 50 MB RAM) availables. Ideal to get started at an extremely low cost.</p>
+          </div><!-- /.col-lg-4 -->
+        </div>        
+      </div>     
+    </div> <!-- / .row -->
+    <div class="row">
+      <div class="col-12 col-auto text-center mt-4 mb-2">   
+        <a href="/users/register" class="btn btn-success btn-lg lift">
+          Get started
+        </a>
+      </div> 
+    </div>
+</section>
+
+<%= render 'home/sections/features_header' %>
+<%= render 'home/sections/features_list' %>
+
+<section class="py-8 pt-md-11 pb-md-12 bg-light p-3 p-md-5 border-top">
+  <div class="container">
+    <div class="row justify-content-center">
+      <div class="col-12 col-md-10 col-lg-8 text-center">      
+        <!-- Preheading -->
+        <h6 class="text-uppercase text-success font-weight-bold">
+          we support opensource
+        </h6>
+        <!-- Heading -->
+        <h1 class="font-weight-bold">Free Hosting For<br/><span class="text-secondary">Selected Projects</span></h1>
+        <div class="device-combo device-combo-macbook-iphonex mb-6 mb-md-0">
+          <div class="device device-macbook text-center">              
+            <img src="/images/undraw_open_source.svg" class="img-fluid" alt="...">
+          </div>                          
+        </div>
+        <!-- Text -->
+        <div class="row text-gray-500 mt-5 mb-7 mb-md-9">
```

Comment:
Add a title before the projects, something like:

![image](https://user-images.githubusercontent.com/8007722/71545990-f22eb280-295f-11ea-993b-5588844fa5c8.png)